### ISSUE TITLE
Provide reference to "INCF types"

### ISSUE THREAD

**Issue by yarikoptic** [NeurodataWithoutBorders/nwb-schema#292]
mentioned in the text, e.g. `format_description.rst`, without any reference 
```shell
$> git grep 'INCF type'
core/nwb.file.yaml:      should be stored in standard types (eg, INCF types) and appropriately documented.
docs/format/source/format_description.rst:superset of several INCF types, including signal events, image stacks
orig/schema.json:                    "content": "\n   <p style=\"margin-bottom: 0in; page-break-after: avoid\">The file\n      format is designed around a data structure called a <i>TimeSeries</i>\n      which stores time-varying data. A <i>TimeSeries</i> is a superset of\n      several INCF types, including signal events, image stacks and\n      experimental events. To account for different storage requirements\n      and different modalities, a <i>TimeSeries</i> is defined in a minimal\n      form and it can be extended, or subclassed, to account for different\n      modalities and data storage requirements. When a <i>TimeSeries</i> is\n      extended, it means that the 'subclassed' instance maintains\n      or changes each of\n      the components (eg, groups and datasets) of its parent and may have\n      new groups and/or datasets of its own. The <i>TimeSeries</i> makes\n      this process of defining such pairs more hierarchical.</p>\n    <p style=\"margin-bottom: 0in\"><br>\n    </p>\n    <p style=\"margin-bottom: 0in\">Each <i>TimeSeries</i> has its own HDF5\n      group, and all datasets belonging to a <i>TimeSeries</i> are in that\n      group. The group contains time and data components and users are free\n      to add additional fields as necessary. There are two time objects\n      represented. The first, <i>timestamps</i>, stores time information\n      that is corrected to the experiment's time base (i.e., aligned to a\n      master clock, with time-zero aligned to the starting time of the\n      experiment). This field is used for data processing and subsequent\n      scientific analysis. The second, <i>sync</i>, is an optional group\n      that can be used to store the sample times as reported by the\n      acquisition/stimulus hardware, before samples are converted to a\n      common timebase and corrected relative to the master clock. This\n      approach allows the NWB format to support streaming of data directly\n      from hardware sources.</p>\n      ",
orig/schema.json:                    "description": "Lab-specific and custom scientific analysis of data. There is no defined format for the content of this group - the format is up to the individual user/lab. COMMENT: To facilitate sharing analysis data between labs, the contents here should be stored in standard types (eg, INCF types) and appropriately documented. MORE_INFO: The file can store lab-specific and custom data analysis without restriction on its form or schema, reducing data formatting restrictions on end users. Such data should be placed in the analysis group. The analysis data should be documented so that it is sharable with other labs",
changes on filesystem:                         
```

**Comment by bendichter** [comment]
It looks like this is an old term from NWB 1.0 that hasn't been fixed yet. It looks like we are talking about what we now refer to as `neurodata_types`

**Comment by yarikoptic** [comment]
Ah cool, I can fix that as going through (#291 might grow ;))

**Comment by oruebel** [comment]
Yes, that is left-over from NWB 1.0. If I remember correctly, Fritz and Jeff were also part of an INCF working group on neuro data at some point and I believe that is referring to that. Either way, this can be cleaned up.
